lib/src/utils.dart

```diff
@@ -104,3 +105,84 @@ String getResponseError(Response res) {
 
 Future<String> ownerToAddress(String owner) async => encodeBytesToBase64(
     await sha256.hash(decodeBase64ToBytes(owner)).then((res) => res.bytes));
+
+Uint8List longTo32ByteArray(int long) {
+  // we want to represent the input as a 8-bytes array
+  final byteArray = [
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+  ];
```

Comment:
Is this possible here?
```suggestion
  final byteArray = new List(32);
```

Reply:
From [here](https://api.dart.dev/stable/2.14.4/dart-core/List/List.filled.html), maybe:
```suggestion
  final byteArray = List.filled(32, 0, growable: false);
```

Reply:
We could actually use `List.generate` as we already are looping through the values to generate 'em in the loop below this line :thinking: